src/main/java/Ship/Ship.java

```diff
@@ -10,6 +10,26 @@
 	public Ship(Point pose, int speed) {
 		
 		super(pose);
+		// create ship shape
+		Polygon shipShape = new Polygon();
+		shipShape.addPoint(0, 20);
+		shipShape.addPoint(10, 0);
+		shipShape.addPoint(20, 20);
+		shipShape.addPoint(10, 10);
+		shipShape.addPoint(0, 20);
+		setShape(shipShape);
+		
+		width = 20; // ship widths
+		
+		moveX(pose.x);
+		moveY(pose.y);
+		
+		dx = speed;
+	}
+
+
+	public Ship(Point pose, int speed, int initialHealth) {
+		super(pose);
```

Comment:
It looks like Herr you've created an overloaded function. It looks similar to the constructor with no health, the only change being that health is initially set. Consider making the overloaded constructor call the regular constructor "this(pose, speed)" and then set health to initialhealth. This will clean up the code a bit and avoid duplicate code.